cogniopenapp/test/homeScreen_test.dart

```diff
@@ -0,0 +1,37 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility in the flutter_test package. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:cogniopenapp/main.dart';
+
+void main() {
+  testWidgets('Tests that the application home page loads correctly.', (WidgetTester tester) async {
+    // Build our app and trigger a frrame.
+    await tester.pumpWidget(MyApp());
+    await tester.pumpAndSettle();
+    await tester.pump();
+
+    // Verify that our counter starts at 0.
+    expect(find.text('CogniOpen', skipOffstage: false), findsOneWidget);
+    expect(find.byKey(const Key("GalleryButtonKey"), skipOffstage: false), findsOneWidget);
+    expect(find.byKey(const Key("VirtualAssistantButtonKey"), skipOffstage: false), findsOneWidget);
+    expect(find.byKey(const Key("AudioRecordingButtonKey"), skipOffstage: false), findsOneWidget);
+    //expect(find.byKey(const Key("SearchButtonKey"), skipOffstage: false), findsOneWidget);
```

Comment:
Do we want these commented out? Is it commented out because the functionality doesn't exist yet?

Reply:
Those checks were failing, despite the buttons being on the screen.  I have commented them out for now. Later, once I figure out the problem, I will uncomment them.

Reply:
If they're not working, can you put a comment stating what isn't working and why? Someone might come through and delete commented out code thinking it isn't important, or it might get forgotten.

Reply:
Okay I did that.